File: Recursion/subtract_twoNumber.c
#include <stdio.h>

int subtract(int a, int b) {
    if (b == 0)
        return a;
    else
        return subtract(a - 1, b - 1);
}

int main() {
    int num1, num2;
    printf("Enter the first integer: ");
    scanf("%d", &num1);
    printf("Enter the second integer: ");
    scanf("%d", &num2);
    int result = subtract(num1, num2);
    printf("Difference = %d\n", result);
    return 0;
}

File: Recursion/subtract_twoNumber.py
def subtract_recursive(a, b):
    if b == 0:
        return a
    else:
        return subtract_recursive(a - 1, b - 1)

num1 = int(input("Enter the first integer: "))
num2 = int(input("Enter the second integer: "))
result = subtract_recursive(num1, num2)
print("Difference =", result)
